### PR TITLE
Create EOL notice

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Project End of Life'd!
+Deployment of CrowdStrike into Red Hat OpenShift originally began here, as a daemonset, but has evolved into a Helm chart. Please see [https://github.com/CrowdStrike/falcon-helm](https://github.com/CrowdStrike/falcon-helm) for active development on a kubernetes distribution-agnostic deployment method!
+
+This project has been moved into Archive state, and will be deleted on/near 1-JUNE-2021.
```
Comment (shawndwells) [inline README.md:4]:
```suggestion
This project has been moved into Archive state and will be deleted on/near 1-JUNE-2021. 
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# Project End of Life'd!
+Deployment of CrowdStrike into Red Hat OpenShift originally began here, as a daemonset, but has evolved into a Helm chart. Please see [https://github.com/CrowdStrike/falcon-helm](https://github.com/CrowdStrike/falcon-helm) for active development on a kubernetes distribution-agnostic deployment method!
```
Comment (redhatrises) [inline README.md:2]:
```suggestion
Deployment of CrowdStrike into Red Hat OpenShift originally began here, as a daemonset, but has evolved into a Helm operator and Helm Chart. Please see [https://github.com/CrowdStrike/falcon-helm](https://github.com/CrowdStrike/falcon-helm) for active development on a kubernetes distribution-agnostic deployment method!
```

**Comment by shawndwells** [comment]
@redhatrises good catch. Changes made!

**Comment by redhatrises** [comment]
LGTM

**Comment by shawndwells** [comment]
Then merge! :)

**Comment by redhatrises** [comment]
Can't 😆

**Pull request merged**